ddpui/core/charts/echarts_config_generator.py

```
@@ -40,17 +40,33 @@ def generate_number_config(data: Dict[str, Any], customizations: Dict[str, Any]
 
         # Create a custom configuration for displaying a single metric
         # Using a gauge series with custom text display
+        # Center all content vertically as a unit using offsetCenter for positioning
+
+        # Calculate offset to position chart title above the number
+        # Title should be positioned above the number value
+        title_offset_y = -(font_size // 2 + 30)  # Position title above number
```

Ruff (0.14.2)

Local variable `title_offset_y` is assigned to but never used

Remove assignment to unused variable `title_offset_y`

(F841)